.circleci/ansible/get_binaries.sh

```diff
@@ -7,7 +7,7 @@ RELEASE=${RELEASE:-false}
 # look at docker hub for latest tag for neon docker image
 if [ "${RELEASE}" = "true" ]; then
     echo "search latest relase tag"
-    VERSION=$(curl -s https://registry.hub.docker.com/v1/repositories/neondatabase/neon/tags |jq -r -S '.[].name' | grep release | sed 's/release-//g' | tail -1)
+    VERSION=$(curl -s https://registry.hub.docker.com/v1/repositories/neondatabase/neon/tags |jq -r -S '.[].name' | grep release | sed 's/release-//g' | grep '^[0-9]*$' | sort -n | tail -1)
```

Comment:
Note that I don't see any images containing 'release' currently.

Reply:
yes currently there are no neon images with release prefix (they are in  `zenithdb` docker account) but they will appear when we merge to release.